merge.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-#!/usr/bin/env python
+# !/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python
```

That space is wrong, see https://en.wikipedia.org/wiki/Shebang_%28Unix%29.